tests/test_doctor.py

```diff
@@ -40,8 +40,49 @@ def test_keys_check(tmp_config: TemporaryConfiguration) -> None:
         "author": "Sanger, F. and Nicklen, S. and Coulson, A. R.",
         })
 
+    error1, error2 = keys_exist_check(doc)
+    assert error1.payload == "ref" or error2.payload == "ref"
+    assert error1.payload == "author_list" or error2.payload == "author_list"
+
+
+def test_keys_check_authors(tmp_config: TemporaryConfiguration,
```

Comment:
Small note: removed a bunch of tests in here since they seemed to be checking different author separators and formats. That was already checked in #654.